src/QMCTools/Eshdf_gpaw.py

```diff
@@ -0,0 +1,281 @@
+######################################################################################
+## This file is distributed under the University of Illinois/NCSA Open Source License.
+## See LICENSE file in top directory for details.
+##
+## Copyright (c) 2021 QMCPACK developers.
+##
+## File developed by: Juha Tiihonen, tiihonen@iki.fi, University of Jyvaskyla
+##
+## File created by: Juha Tiihonen, tiihonen@iki.fi, University of Jyvaskyla 
+#######################################################################################
+
+from numpy import array,pi,dot,real,imag,concatenate,unique,zeros
```

Comment:
both Eshdf_gpaw and gpaw4qmcpack need a brief documentation at the beginning about what they do. Probably just need one line.